examples/bri-3/blockchain/blockchain.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+
+@Injectable()
+export class BlockchainService implements OnModuleInit {
```

Comment:
should this service be outside of module? maybe blockchain module with stuff like blockchainService, merkleTreeService and so on

Reply:
I was debating this same question. I placed the blockchain service outside of the ZKP component in case other modules also need to access the blockchain without using the ZKP. What are your thoughts?

Reply:
it is still possible if its part of the module, module would export service and then other modules could still import module and inject service

Reply:
Yes, should be a module in itself